### PR TITLE
return None if not installed

### PR DESCRIPTION
**Pull request by luizoti**
This is a simple modification.

1. removed cmd command error.
2. If the browser is not installed, return None (message is still UNKNOW)
3. Asks if the browser is installed.

### CODE REVIEW

**Comment by aleksandr-kotlyar** [comment]
@luizoti what if just make disable WDM logs?

**Comment by luizoti** [comment]
> what if just make disable WDM logs? Would it be better than making this overhead on webdriver-manager

From my point of view, if there is an option that turns off error messages and another that turns off information messages, there is no problem, I include this optional argument exactly because I didn't see any sense in the error message, since just printing the message didn't make it clear what happened (so I added a question too).

**Comment by aleksandr-kotlyar** [comment]
@luizoti thank you! You made me think to remove this error message at all. Who wants - can go to sources and debug.
Because the implementation of disabling this error message is a huge overhead.

**Comment by aleksandr-kotlyar** [comment]
@luizoti You can make a PR with removing `"with the command {cmd}"` from this logging message
```python
        log(f'Could not get version for {browser_type} with the command: {cmd}')
```

**Comment by luizoti** [comment]
I updated, removed the cmd and asked the question whether the browser is installed or not in place.

File: webdriver_manager/utils.py [hunk 1]
```diff
@@ -241,12 +240,13 @@ def get_browser_version_from_os(browser_type=None):
     version = read_version_from_cmd(cmd, pattern)
 
     if not version:
-        log(f'Could not get version for {browser_type} with the command: {cmd}')
+        log(f'Could not get version for {browser_type}, is {browser_type} installed?')
 
-    current_version = version.group(0) if version else 'UNKNOWN'
-
-    log(f"Current {browser_type} version is {current_version}")
+    current_version = version.group(0) if version else None
```
Comment (aleksandr-kotlyar) [inline webdriver_manager/utils.py:245]: leave this part like it was before please

Comment (aleksandr-kotlyar) [inline webdriver_manager/utils.py:245]: leave this part like it was before please

File: webdriver_manager/utils.py [hunk 2]
```diff
@@ -241,12 +240,13 @@ def get_browser_version_from_os(browser_type=None):
...
 
+    log(f"""Current {browser_type} version is {
+            current_version if current_version else 'UNKNOW'
+        }""")
```
Comment (aleksandr-kotlyar) [inline webdriver_manager/utils.py:249]: leave this part like it was before please (2)

Comment (aleksandr-kotlyar) [inline webdriver_manager/utils.py:249]: leave this part like it was before please (2)

File: webdriver_manager/utils.py [hunk 3]
```diff
@@ -241,12 +240,13 @@ def get_browser_version_from_os(browser_type=None):
     version = read_version_from_cmd(cmd, pattern)
 
     if not version:
-        log(f'Could not get version for {browser_type} with the command: {cmd}')
+        log(f'Could not get version for {browser_type}, is {browser_type} installed?')
```
Comment (aleksandr-kotlyar) [inline webdriver_manager/utils.py:243]: Only this change means.

Comment (aleksandr-kotlyar) [inline webdriver_manager/utils.py:243]: Only this change means.

**Review by aleksandr-kotlyar — CHANGES_REQUESTED** [review]

**Review by aleksandr-kotlyar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**